.github/workflows/fork-cherry-pick.yml

```diff
@@ -1,3 +1,4 @@
+name: Cherry pick fork PR
```

Comment:
please disable and leave a comment asking forks to enable. We should also add it to some docs but that can come later

Reply:
Yep! I believe this is currently disabled already on cosmos-sdk/main - will add a comment though to tell forks to enable

Reply:
can you point me to where its disabled. I think it will run on this repo as well

Reply:
I believe we can do it manually via the GH interface: https://github.com/cosmos/cosmos-sdk/actions/workflows/fork-cherry-pick.yml